charts/redpanda/values.go

```diff
@@ -954,10 +955,10 @@ func (s *Sidecars) AdditionalSidecarControllersEnabled() bool {
 }
 
 type Listeners struct {
-	Admin          AdminListeners          `json:"admin" jsonschema:"required"`
-	HTTP           HTTPListeners           `json:"http" jsonschema:"required"`
-	Kafka          KafkaListeners          `json:"kafka" jsonschema:"required"`
-	SchemaRegistry SchemaRegistryListeners `json:"schemaRegistry" jsonschema:"required"`
+	Admin          ListenerConfig[NoAuth]                    `json:"admin" jsonschema:"required"`
```

Comment:
Admin API has an option to enable Basic Auth headers authorization. It is cluster property that needs to coexist with superusers property.  

https://docs.redpanda.com/current/reference/properties/cluster-properties/#admin_api_require_auth

https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#secure-the-admin-api

Reply:
Right! `NoAuth` indicates that the listener, as defined in the redpanda.yaml, does not support an `authentication_method` field.